com.unity.template-hd/Packages/com.unity.template.hd/package.json

```diff
@@ -7,9 +7,47 @@
   "unity": "2022.1",
```

Comment:
Shouldn't it be 21.2 ?

Reply:
PR is for master a backport to 21.2 will be perform after